### DISCUSSION TITLE
Conditional validation schema between null and rules.

### DISCUSSION THREAD

**Discussion by Saad-Bashar** [jaredpalmer/formik#3432] [General]
I have a form where users can submit a draft and at the same time save the form while submitting the draft I want to make all the validation schema to be null or empty so that it does not validate anything and use validation schema when saving the form. 

How can I conditionally change the schema in my case?

**Comment by fidaay** [comment]
Just make an arrow function like:

`validationSchema={() => {
   if(condition) {
      return
   }
  else {
      return
   }
}`
